scheduler/src/cook/components.clj

```diff
@@ -217,6 +217,15 @@
         (h req)
         (auth-fn req)))))
 
+(defn conditional-rate-limit
```

Comment:
I modeled this after `conditional-auth-bypass` (line 210 above). I debated making it more customizable, like maybe passing a dict of endpoint -> custom limit, but I decided that we can do that if we ever need to add another custom limit instead of fidgeting with it now.

Reply:
sounds good to me